src/EditorFeatures/CSharpTest/UsePatternMatching/CSharpAsAndNullCheckTests_FixAllTests.cs

```diff
@@ -27,7 +27,7 @@ int M()
         {
         }
 
-        if ((a = o as string) != null)
+        if ((a = o as string) == null)
```

Comment:
❗️ (Unless not possible) it would be good to retain the original `if` statement as well as the new one.

Reply:
I just changed this test to cover negative case.

Reply:
:arrow_right: Sounds good, thanks